coalib/parsing/filters/UseStdinFilter.py

```diff
@@ -0,0 +1,31 @@
+def use_stdin_filter(bears, use_stdin='True'):
+    bears = filter_bears_by_stdin(bears)
+    return bears
+
+
+def filter_bears_by_stdin(bears, use_stdin='True'):
+    local_bears, global_bears = bears
+    local_bears = filter_section_bears_by_stdin(local_bears, use_stdin)
+    global_bears = filter_section_bears_by_stdin(
+        global_bears, use_stdin)
+    return local_bears, global_bears
+
+
+def filter_section_bears_by_stdin(bears, use_stdin='True'):
+    """
+    Filters the bears by use_stdin.
+
+    :param bears:       The dictionary of the sections as keys and list of
+                        bears as values.
+    :param use_stdin:   Value indicating whether filtered bears
+                        should use stdin
+    :return:            New dictionary with filtered out bears with
+                        use_stdin value specified in param.
+    """
+    new_bears = {}
+    stdin_values = set(stdin.lower() for stdin in use_stdin)
+    for section in bears.keys():
+        new_bears[section] = tuple(
+            bear for bear in bears[section]
+            if {stdin.lower() for stdin in bear.use_stdin} & stdin_values)
```

Comment:
you shouldn't filter for `use_stdin`, this is an implementation detail of the bears. `use_stdin` is stored inside a dict of the bear decorated by `@linter`, but I'm not sure if it's scoped and thus not accessible. But like said, I wouldn't implement a filter for this^^ Maybe for the executable of a linter this might be useful, but not all bears are linter bears ;)